projectile.el

```diff
@@ -911,10 +937,13 @@ This variable is reset automatically when Projectile detects that
 the `buffer-file-name' has changed. It can also be reset manually
 by calling `projectile-reset-cached-project-name'.")
 
-(defvar-local projectile-cached-buffer-file-name nil
```

Comment:
I just moved it up earlier in the file so it is declared before we now first reference it.